source/distributed/grid_refinement.cc

```diff
@@ -186,6 +186,67 @@ namespace
           cell->clear_coarsen_flag();
         }
   }
+
+
+
+  /**
+   * Fixed fraction algorithm without a specified vector norm.
+   *
+   * Entries of the criteria vector and fractions are taken as is, so this
+   * function basically evaluates norms on the vector or its subsets as
+   * l1-norms.
+   */
+  template <int dim, typename Number, int spacedim>
+  void
+  refine_and_coarsen_fixed_fraction_l1_norm(
```

Comment:
This is an internal function, right? If so, put it into an anonymous namespace.

Reply:
Also, if it's an internal function, just use the "traditional" order of template arguments: dim, spacedim, Number

Reply:
Actually, it is already in an anonymous namespace.

I took the order of template arguments from the corresponding functions in `GridRefinement` and `parallel::distributed::GridRefinement`. I will follow your suggestion.